### ISSUE TITLE
Folder Scroll Direction

### ISSUE THREAD

**Issue by armc** [Maestroschan/appfolders-manager-gnome-extension#61]
Hello,
I appreciate the work on the extension; however, I find that I get a little frustrated with it since the folders do not follow the system scroll direction.

I use natural scrolling (Moving of "Page") instead of scrolling (Moving of "View Port").  The inconsistency leads to some frustration when using folders.

Is there a setting that I can change or one that can be added to make the folder scroll according to system settings?

**Comment by maoschanz** [comment]
I'm not sure to understand your issue, could you rephrase it please ?

**Comment by armc** [comment]
I must apologize.  Previously (for what ever reason) scrolling was reversed in an expanded folder.  It appears to function correctly.

What I experienced previously was:
(Use Two Finger Scroll):  Push Up to go down the application grid pages.  Click to expand a Folder Group.  Then I had to Pull Down to go down the expanded folders contents.

I don't know why it has corrected itself but I apologize.  I have experienced this before so today's issue led me to post.  It was not the only time I have had the problem.

I don't use the "Show Applications" grid a lot (usually type to select app), but sometimes I do to explore a less commonly used app.

Thanks for the quick response.
